/*try{
    console.log("awal blok try");
    console.log(a);
    console.log("akhir blok try");
}
catch(error){
    console.log(error.name);
    console.log(error.message);
    console.log(error.stack);
}
finally{
    console.log("akan tetap di eksekusi");//akan tetap dijalankan
}*/

//Throwing errors
